examples/tumbleweed/Dockerfile

```diff
@@ -77,7 +85,10 @@ RUN echo "PermitRootLogin yes" > /etc/ssh/sshd_config.d/rootlogin.conf
 ADD 05_network.yaml /system/oem/05_network.yaml
 
 # Generate initrd with required elemental services
-RUN elemental --debug init --force
+RUN elemental --debug init --force elemental-rootfs,elemental-sysroot,grub-config,elemental-setup,dracut-config,cloud-config-essentials,boot-assessment,cloud-config-defaults,selinux,selinux-example
+
+# Load selinux module
+RUN semodule -i /usr/share/elemental/selinux/elemental.pp
```

Comment:
We could consider loading all `.pp` files in this directory during `elemental-init`